test/unit/spec/EnrollChoices_spec.js

```diff
@@ -522,6 +545,97 @@ function (Okta, OktaAuth, Util, EnrollChoicesForm, Beacon, Expect, FactorUtil, R
         });
       });
 
+      Expect.describe('Grace period', function () {
+        beforeEach(function () {
```

Comment:
should we install/uninstall  clock so we just restrict the jasmine clock behavior to this block  ?

Reply:
Filed OKTA-241179 to look into this